src/metapenta/model/MetaPenta.java

```diff
@@ -28,6 +30,11 @@ public void describeMetabolicNetwork(String outFilePrefix) throws Exception {
         petriNet.describeMetabolicNetwork(outFilePrefix);
     }
 
+    @Override
+    public void sourcesFinder(String outFilePrefix) throws Exception {
+        //petriNet.getSources(outFilePrefix);
+        
+    }
```

Comment:
Can you remove the source finder logic of this PR? Lets keep this PR just for Balance Reactions